src/esp/sensor/CameraSensor.cpp

```diff
@@ -154,10 +154,12 @@ bool CameraSensor::drawObservation(sim::Simulator& sim) {
     // SensorType is Color, Depth or any other type
     draw(sim.getActiveSceneGraph(), flags);
 
-    // include DebugLineRender in Color sensors
     if (cameraSensorSpec_->sensorType == SensorType::Color) {
+      // include HBAO in Color sensors (only if enabled for render target)
+      renderTarget().tryDrawHbao();
```

Comment:
Currently, renderTarget owns and uses an `HBAOHelper`, including giving access to OpenGL framebuffer and depth-buffer indices. This should probably be changed, i.e. keep HBAOHelper out of RenderTarget.cpp altogether.

My preference would be for `CameraSensor` to own HBAOHelper and for HBAOHelper to use our renderTarget's public API as necesseary, e.g. `bind` and `getDepthTexture`.